commit f1f068e72d32c6df8c5d8f045b31611d632763f7
Author: karog
Date:   Fri Jun 1 17:31:26 2012 -0400

    modified:   class_autoload.php
    Moved search of include_path down since on OS X it prematurely found
    case indpendent matching files.

diff --git MythWeb/includes/class_autoload.php MythWeb/includes/class_autoload.php
index 8ae760c..36b70d9 100644
--- includes/class_autoload.php
+++ includes/class_autoload.php
@@ -14,24 +14,24 @@
     function autoload($className) {
         global $Path;
         $className = str_replace('_', '/', $className);
-        if (file_exists("classes/$className.php")) {
+        if (file_exists("classes/$className.php"))
             include_once "classes/$className.php";
-            return true;
-        }
-        foreach (explode(PATH_SEPARATOR, ini_get('include_path')) as $path) {
-            if (!file_exists("$path/$className.php"))
-                continue;
-            include_once "$path/$className.php";
-            return true;
-        }
-        if (file_exists(modules_path.'/'.module."/classes/$className.php"))
+        elseif (file_exists(modules_path.'/'.module."/classes/$className.php"))
             include_once modules_path.'/'.module."/classes/$className.php";
         elseif (file_exists(modules_path.'/'.$Path[1]."/classes/$className.php"))
             include_once modules_path.'/'.$Path[1]."/classes/$className.php";
         elseif (file_exists(modules_path.'/'.$Path[0]."/classes/$className.php"))
             include_once modules_path.'/'.$Path[0]."/classes/$className.php";
-        else
+        else {
+            foreach (explode(PATH_SEPARATOR, ini_get('include_path')) as $path) {
+                if (!file_exists("$path/$className.php"))
+                    continue;
+                include_once "$path/$className.php";
+                return true;
+            }
             return false;
+        }
+        return true;
     }
 
     spl_autoload_register('autoload');
